toolchains/jp_workspaces/requirements.txt

```diff
@@ -1,4 +1,5 @@
-setuptools==70.2.0
-numpy<2.0.0
+--index-url https://pypi.jetson-ai-lab.dev/jp6/cu126
```

Comment:
Id like to add this index to the pyproject.toml to support uv as a build tool which I think has the best UX but there's likely some important details we need to think about. cc: @lanluo-nvidia for later